packages/react-native-lockdown/src/repair.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Repair JS intrinsics
+ *
+ * The Hermes (or vanilla) SES shim is loaded as the first Metro polyfill, so
+ * {@link repairIntrinsics} can be called.
+ *
+ * Called with lockdown options ideal for React Native apps.
+ *
+ * {@link lockdown} itself is not called to allow for vetted shims as Metro
+ * polyfills before hardening (e.g. RN JS polyfills).
```

Comment:
```suggestion
 * polyfills before hardening (e.g. RN JS polyfills).
 *
 * @packageDocumentation
```

omitting the tag means you are documenting the variable on L16

Reply:
note to self: this is a tsdoc [thing](https://tsdoc.org/pages/tags/packagedocumentation/)

(that typedoc can [use](https://typedoc.org/documents/Tags._packageDocumentation.html))